tests/unit/fileserver/test_gitfs.py

```diff
@@ -252,8 +246,8 @@ class GitFSTestFuncs(object):
     2. Do *NOT* move the gitfs.update() into the setUp.
     """
 
-    @skipIf(True, "SLOWTEST skip")
-    @skipIf(True, "SLOWTEST skip")
+    @slowTest
+    @slowTest
```

Comment:
I guess I really wanted to skip this and the next one, too :joy:

Reply:
Fixed these in several files that had them.